testing/trino-tests/src/test/java/io/trino/tests/TestMockConnector.java

```diff
@@ -61,6 +62,12 @@ protected QueryRunner createQueryRunner()
                                     return ImmutableList.of(new ColumnMetadata("nationkey", BIGINT));
                                 })
                                 .withGetTableHandle((session, tableName) -> new MockConnectorTableHandle(tableName))
+                                .withGetMaterializedViewProperties(() -> ImmutableList.of(
```

Comment:
Do we have a test coverage for `ALTER MV` with mock connector? Could we have one?

Reply:
I have added a new test class for `SetPropertiesTask`.